scripts/variantstore/wdl/GvsQuickstartHailIntegration.wdl

```diff
@@ -139,6 +139,16 @@ task CreateAndTieOutVds {
         pip install pytest
         ln -s ${WORK}/joined.mt .
         pytest ./gvs_vds_tie_out.py
+
+
+        # Copy down our VDS validation script -- it should stay constant so we dont need it to be branch specific
```

Comment:
🙁 don't love this constraint but fixing it is certainly out of scope for this PR